horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/AlertingServiceImpl.java

```diff
@@ -1030,8 +1031,8 @@ public void checkMissingDataset() {
    public List<DatapointLastTimestamp> findLastDatapoints(LastDatapointsParams params) {
       Query query = em.createNativeQuery(FIND_LAST_DATAPOINTS)
             .unwrap(NativeQuery.class)
-            .setParameter(1, Util.parseFingerprint(params.fingerprint), JsonNodeBinaryType.INSTANCE)
-            .setParameter(2, params.variables, IntArrayType.INSTANCE);
+            .setParameter(1, Util.parseFingerprint(params.fingerprint), new CustomType<>(new JsonBinaryType(), new TypeConfiguration()))
+            .setParameter(2, params.variables, IntArrayType.INT_ARRAY);
```

Comment:
I noticed that we're not checking this endpoint in our tests. I have not verified that the IntArrayType works...

Reply:
we should add a test for this